tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import os
+import pytest
+
+from freezegun import freeze_time
```

Comment:
What module are you using for this? As far as I know there are three: `freezegun`, `pytest-freezegun`, and `pytest-freezer`. I believe `pytest-freezer` is the best/most up to date.

Reply:
I'm using freezegun -- I agree that the others are more up to date but this is the only one I got working in conftest :/ Otherwise I could define it manually using the other packages in each test file. Willing to make the change tho!